### PR TITLE
Get List of Accounts

### LINKED ISSUE #7: Get list of accounts [2]

### PR DESCRIPTION
**Pull request by klampa10**
Closes #7.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,22 +1,30 @@
 /* eslint-disable no-console */
 import express, { Application, Request, Response } from 'express';
 import mongoose from 'mongoose';
+/* import User, { IUser } from './models/user'; */
```
Comment (carldegs) [inline src/index.ts:4]: Remove unused code

File: src/index.ts [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
 db.on('error', console.error.bind(console, 'connection error:'));
 db.once('open', () => {
   console.log('connected to db!');
 });
 
 // Application routing
-app.use('/', (req: Request, res: Response) => {
-  res.status(200).send({ data: 'Hello from the other side' });
+app.use('/', (req: Request, res: Response, next) => {
+  // res.status(200).send({ data: 'Hello from the other side' });
```
Comment (carldegs) [inline src/index.ts:23]: Do not comment out. Maybe just change the route to `/ping` .

**Review by carldegs — COMMENTED** [review]
Very minor changes

File: src/index.ts (hunk 2 above)
Comment (klampa10) [reply src/index.ts]: Fixed.

File: src/index.ts (hunk 1 above)
Comment (klampa10) [reply src/index.ts]: Fixed.

**Pull request merged**